pkg/registry/rbac/rest/storage_rbac.go

```diff
@@ -65,7 +68,10 @@ type RESTStorageProvider struct {
 	Authorizer authorizer.Authorizer
 }
 
-var _ genericapiserver.PostStartHookProvider = RESTStorageProvider{}
+var (
+	_           genericapiserver.PostStartHookProvider = RESTStorageProvider{}
+	parallelism                                        = 5
```

Comment:
how was this value selected? what was the impact of lower and higher values on startup time (or a value of `1` matching the current behavior)?